src/main/java/dao/TaskDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+package dao;
+
+import configuration.Setting;
+import model.Task;
+import storage.FileListStorage;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
```

Comment:
I think your indentation levels are too big. Should be 4 spaces.

Reply:
Yes, you can change your IDE / text editor's settings to reconfigure your tab to be of 4 spacings.

Reply:
Yup, resolved, forgot to change spaces to tabs